README.md

```diff
@@ -1,133 +1,40 @@
-# iNaturalist-PythonProjectTemplate
+# iNaturalist GeoModel Annotator Project
 
 Code repository for 2024 Data Science for the Common Good project with iNaturalist. 
 
-Collaborators: Angela Zhu, Paula Navarrete, Sergei Pogorelov
+Collaborators: Angela Zhu, Paula Navarrete, Sergei Pogorelov, Ozzie 
 
 This template draws a lot of inspiration from [Cookiecutter Data Science](https://drivendata.github.io/cookiecutter-data-science/). Please read their awesome explanations!
 
 # Getting Started
-## Installation for development
+## :hatched_chick: Installation for local development
 Make sure you update your local branch to the latest.
 
 ```
 $ conda create -n inatator python=3.9
 $ conda activate inatator
 ```
 
-### React components
-1) Navigate to `src/frontend`, run `npm i` that will install js libraries needed for react.
+### React Components
+1) Navigate to `src/frontend`
+2) Run `npm i` that will install js libraries needed for react.
 
-### Backend and SINR
-- Check if you have `poetry`, this is needed because poetry is a tool that finds all compatible library versions while downloading them, otherwise we would die
-- To install poetry go to https://python-poetry.org/docs/
-- It will need you to install pipx, pip on steroids, follow the official pipx instructions to install
-- You don't have to run the command with --global option, the last command
-- Steps will differ slightly based on your OS
+### Backend and SINR Dependency Installation
+1. Check if you have `poetry`, this is needed because poetry is a tool that finds all compatible library versions while downloading them, otherwise we would die
+  - To install poetry go to https://python-poetry.org/docs/
+  - It will need you to install pipx, pip on steroids, follow the official pipx instructions to install
+  - You don't have to run the command with --global option, the last command
+  - Steps will differ slightly based on your OS
+  - After pipx is installed you can install poetry from their instructions, just one line
+2. Check poetry by just typing `poetry` into terminal, it should recognize and list options etc
 
-- After pipx is installed you can install poetry from their instructions, just one line
-- Check poetry by just typing `poetry` into terminal, it should recognize and list options etc
+3. Now at the project root you can run `poetry install` which will install all packages required for now
+  - To double check everything is working as intended, navigate to `src/sinr/models.py` write a function that prints something. Now, navigate to `src/backend/app` and create a temporary python file that you will delete later, in the file enter `from sinr.models import *`, then call the function you just created from that file, make sure it prints correctly and does not give errors.
 
```

Comment:
If this PR is still open, please add the following:

### Running applications with Docker
1. Install Docker if you haven't already
2. Navigate project root
3. Run `docker-compose up --build`
  - For the first build it may take a while
4. Make sure the default application is running as expected by going to urls for react and server
5. During development, you can stop contianers with ctrl+c or using the Docker app
6. If you want to start the application again, run `docker-compose up`
7. If you change a docker configuration file, run `docker-compose up --build`

## Code Standards
1. Use Docstrings, for some functions just a one-linet is fine, but for more complicated functions include multi-line documentation that explains the function simply, has information about arguments, and has details about the output.
2. Module Docstrings, include a short description of module and functions inside the module.
3. Use a formatter if possible, **black** formatter has support for vscode and is decent